File: apzkr-pzpi-21-5-voropaiev-vladyslav/Task3-WebClient/src/app/core/models/pond.ts
export enum FishSpecies {
    Bass,
    Carp,
    Salmon,
    Catfish,
    Tilapia,
    Pike,
    Bluegill,
    Crappie,
    Sturgeon,
    Trout
}

export interface Pond {
    id: string;
    fishSpecies: FishSpecies;
    fishPopulation: number;
    feedingScheduleId: string | undefined;
    createdAt: Date;
}

File: apzkr-pzpi-21-5-voropaiev-vladyslav/Task4-MobileClient/Domain/Entities/Pond.cs
namespace Domain.Entities;

public class Pond
{
    public Guid Id { get; set; }
    
    public string FishSpecies { get; set; } 
    
    public int FishPopulation { get; set; }
    
    public FeedingSchedule FeedingSchedule { get; set; }
}